mcgill-augmented-reality-backend/src/main/java/mcgillar/backend/controllers/user/AppUserInfoController.java

```diff
@@ -23,6 +24,25 @@ public class AppUserInfoController {
     private AppUserInfoService appUserInfoService;
     private AppUserService appUserService;
 
+    
+    /**
+     * 
+     * @param authentication
+     * @return all user info
+     * 
+     * Return all user info for given user
+     */
+    @GetMapping("all-user-info")
```

Comment:
Right now, on postman, the endpoint getting used is just "user-info" written by Matt. Looks like frontend is expecting that to be the endpoint. I agree with it, we don't need to be too specific with this endpoint cuz we wont have a call requesting specific user info